releasenotes/notes/add-support-for-otel-metrics-42396aa1550e621c.yaml

```
@@ -0,0 +1,3 @@
+features:
+  - |
+    otel: This adds support for exporting OpenTelemetry Metrics API. Notably, the application must include its own OTLP metrics exporter.
```

Should we also call out the DD_METRICS_OTEL_METRICS_ENABLED config?  We should document it here: https://github.com/DataDog/dd-trace-py/blob/v3.10.1/docs/configuration.rst (or in the Datadog common configurations page or both). 

```suggestion
    otel: Adds support for exporting OTLP metrics via the OpenTelemetry Metrics API. Notably, the application must include its own OTLP metrics exporter.
```

---

I've updated this in 2b9ba91ee7d1454eb7c49d502ca2bfa4cb89fd1e